docs-chef-io/content/server/upgrade_server_ha_v2.md

```diff
@@ -25,20 +25,35 @@ This documentation describes the process for upgrading a high availability Chef
 
 These instructions cover the process of upgrading a Chef Backend cluster. Please use to the version of Chef Backend already running on your system and your target upgrade version.:
 
+## Prerequisite
```

Comment:
I think these changes are interesting, but they're slightly out of scope. I'm going to create an 'Upgrade` section in the nav and cut this page into several smaller pages. I'd like to look at this proposed change after the new navigation is in place.

Reply:
Leaving these in for now, can be updated as needed in the future.